test/load/test_load_tx_ethbridge_burn.py

```diff
@@ -22,30 +23,97 @@
+    transfer_table = build_transfer_table()
+    amount_per_tx = 1000100101
+
+    _test_load_tx_ethbridge_lock_burn(ctx, amount_per_tx, transfer_table, rowan_token_address, isRowan=True)
+
+# test single sif account burn erc20 to multiple ethereum accounts
+def test_single_sif_to_multiple_eth_account_burn_erc20(ctx: test_utils.EnvCtx):
+    # deploy an erc20 contract
+    token_decimals = 18
+    total_amount = 10 ** 28
+    token_data: test_utils.ERC20TokenData = ctx.generate_random_erc20_token_data()
+    erc20_sc = ctx.deploy_new_generic_erc20_token(token_data.name, token_data.symbol, token_decimals)
+    # mint token to operator account
+    ctx.mint_generic_erc20_token(erc20_sc.address, total_amount, ctx.operator) 
+    ctx.advance_blocks()
+
+    transfer_table = build_transfer_table()
+    amount_per_tx = 1000100101
 
-def test_load_tx_ethbridge_burn_short(ctx: test_utils.EnvCtx):
-    _test_load_tx_ethbridge_burn(ctx, 123456 * eth.GWEI, [[2, 2], [2, 2]])
+    _test_load_tx_ethbridge_lock_burn(ctx, amount_per_tx, transfer_table, erc20_sc.address)
 
+# test single sif account burn ceth to multiple ethereum accounts
+def test_single_sif_to_multiple_eth_account_burn_eth(ctx: test_utils.EnvCtx):
+    transfer_table = build_transfer_table()
+    amount_per_tx = 1000100101
 
-def test_load_tx_ethbridge_burn(ctx: test_utils.EnvCtx):
+    _test_load_tx_ethbridge_lock_burn(ctx, amount_per_tx, transfer_table, None)
+
+# short test to verify eth burn works
+def test_load_tx_ethbridge_burn_eth_short(ctx: test_utils.EnvCtx):
+    transfer_table = [[2, 2], [2, 2]]
+    amount_per_tx = 1000100101
+    _test_load_tx_ethbridge_lock_burn(ctx, amount_per_tx, transfer_table, None)
+
+# test multiple sif accounts burn ceth to multiple ethereum accounts
+def test_load_tx_ethbridge_burn_eth(ctx: test_utils.EnvCtx):   
     # Matrix of transactions that we want to send. A row (list) in the table corresponds to a sif account sending
     # transactions to eth accounts. The numbers are transaction counts, where each transaction is for amount_per_tx.
     # Each sif account uses a dedicated send thread.
     transfer_table = [
-        [100, 100, 100],
-        [100, 100, 100],
-        [100, 100, 100],
-        [10, 20, 30],
-    ]
-
-    amount_per_tx = 123456 * eth.GWEI
-
-    _test_load_tx_ethbridge_burn(ctx, amount_per_tx, transfer_table)
+            [100, 100, 100],
+            [100, 100, 100],
+            [100, 100, 100],
+            [10, 20, 30],
+        ]
+    amount_per_tx = 1000100101
+    _test_load_tx_ethbridge_lock_burn(ctx, amount_per_tx, transfer_table, None)
+
+def _test_load_tx_ethbridge_lock_burn(ctx: test_utils.EnvCtx, amount_per_tx: int, 
+    transfer_table: List[List[int]], token_address: Address, isRowan: bool = False, randomize: bool = None):
+    # rowan is natvie token, denom not from contract in Ethereum
+    if isRowan:
+        token_denom = rowan
+    elif token_address == None:
```

Comment:
This test is used in multiple places in the code, and it is not necessarily obvious that `token_address == None` means that the token being transacted is CETH, except rigt here.

The code below that uses this test could use instead `token_denom == ctx.ceth_symbol` if such things are comparable like that. Else it would be convenient to set a `ceth_flag = true` upon discovering that `token_address == None`, and then refer to that flag where there's code that depends on the token being CETH.

Another possibility is to simply pass `token_denom` as an argument, and then simplify the function's signature such that instead of receiving both a `token_address` and a `isRowan` argument, that it may only have a `token_denom` argument, as long as `token_address` or `isRowan` can be extracted as internal flags from `token_denom`, when necessary.